lib/fizzy/execute.cpp

```diff
@@ -830,8 +830,8 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, const Value* args,
         }
         case Instr::i32_eqz:
         {
-            const auto value = stack.pop().as<uint32_t>();
-            stack.push(uint32_t{value == 0});
+            const auto value = stack.top().as<uint32_t>();
+            stack.top() = uint32_t{value == 0};
```

Comment:
Why keep `value` as a variable here when in `i64_eqz` it is a single line?

Reply:
Because it was like that.

Reply:
How stuff was before and how it is now didn't stop you here: https://github.com/wasmx/fizzy/pull/578/files#diff-da8232128fc88a65268ab30bf8e8abdcL1409 :wink: